docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -0,0 +1,1001 @@
+[`fake_provider.FakeBackendV2LegacyQubitProps`](/api/qiskit/0.46/qiskit.providers.fake_provider.FakeBackendV2LegacyQubitProps) | [`fake_provider.GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2)
+[`fake_provider.FakeBackendSimple`](/api/qiskit/0.46/qiskit.providers.fake_provider.FakeBackendSimple) | [`fake_provider.GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2)
+[`fake_provider.ConfigurableFakeBackend`](/api/qiskit/0.46/qiskit.providers.fake_provider.ConfigurableFakeBackend) | [`fake_provider.GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2)
+
+Migration example to the new
+[`GenericBackendV2`](/api/qiskit/qiskit.providers.fake_provider.GenericBackendV2)
+class:
+
+```python
+# Legacy path
+from qiskit.providers.fake_provider import FakeBackend5QV2
+backend = FakeBackend5QV2()
+
+# New path
+from qiskit.providers.fake_provider import GenericBackendV2
+backend = GenericBackendV2(num_qubits=5)
+# note that this class will generate 5q backend with generic
+# properties that serves the same purpose as FakeBackend5QV2
+# but will not be identical.
+```
+
+<span id="other-migration-tips"></span>
+### Other migration tips
+
+* Importing from `qiskit.providers.aer` is no longer possible. Instead, import
+  from `qiskit_aer`, which is a drop-in replacement. To install `qiskit_aer`,
+  run:
+
+    ```bash
+    pip install qiskit-aer
+    ```
+
+* Support for running pulse jobs on backends from
+  [`qiskit.providers.fake_provider`](/api/qiskit/providers_fake_provider) has
+  been removed in Qiskit 1.0.  This is due to Qiskit Aer removing its
+  simulation functionality for such jobs.  For low-level Hamiltonian-simulation
+  workloads, consider using a specialized library such as [Qiskit
+  Dynamics](https://qiskit-extensions.github.io/qiskit-dynamics/).
+
+<span id="qiskit.pulse"></span>
+## qiskit.pulse
+
+<span id="ParametricPulse"></span>
+### ParametricPulse
+
+The `qiskit.pulse.library.parametric_pulses.ParametricPulse` base class and
+pulse library have been superseded by `qiskit.pulse.SymbolicPulse` and the
+corresponding pulse library. `SymbolicPulse` provides better performance,
+flexibility, and [QPY](../qiskit/qpy) support:
+
+Deprecated                       |   Alternative
+---------------------------------|--------------------------------------------
+[`pulse.library.parametric_pulses.ParametricPulse`](/api/qiskit/0.46/qiskit.pulse) | [`qiskit.pulse.SymbolicPulse`](/api/qiskit/qiskit.pulse.SymbolicPulse)
+[`pulse.library.parametric_pulses.Constant`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`pulse.library.parametric_pulses.Drag`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`pulse.library.parametric_pulses.Gaussian`](/api/qiskit/0.46/qiskit.pulse) |  ?
+[`qiskit.pulse.library.parametric_pulses.GaussianSquare`](/api/qiskit/0.46/qiskit.pulse) |  ?
```

Comment:
Unfortunately I don't have the time to look up concrete examples or paths for the alternatives. If we want to merge this as an MVP, I suggest leaving the table out (maybe turn it into a list) and adding it in a follow-up.